ivy/functional/frontends/torch/tensor.py

```diff
@@ -419,6 +419,12 @@ def tril(self, diagonal=0):
     def index_select(self, dim, index):
         return torch_frontend.index_select(self._ivy_array, dim, index)
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16", "float16")}, "torch")
+    def clamp(self, min=None, max=None, *, out=None):
+        if min is not None and max is not None:
+            return torch_frontend.tensor(ivy.array(self._ivy_array).full_like(max))
```

Comment:
this should be done when min > max when both min and max are not None

so this condition is good but it should also have `... and min > max`

to accomodate arrays, the `all` function (torch frontend or ivy version) may have to be used